lib/public/Model.js

```diff
@@ -13,31 +13,32 @@
 
-import { LogsModel } from './views/Logs/LogsModel.js';
-import Tags from './views/Tags/Tags.js';
-import { RunsModel } from './views/Runs/RunsModel.js';
-import Flps from './views/Flps/Flps.js';
-import { EnvironmentModel } from './views/Environments/EnvironmentModel.js';
-import LhcFills from './views/LhcFills/LhcFills.js';
-import RunTypeModel from './views/RunTypes/RunTypeModel.js';
+import { registerFrontLinkListener } from './components/common/navigation/frontLinkListener.js';
 import { ModalModel } from './components/modal/ModalModel.js';
+import { BkpRoles } from './domain/enums/BkpRoles.js';
+import { UserRoleSelectionModel } from './models/UserRoleSelectionModel.js';
+import { detectorsProvider } from './services/detectors/detectorsProvider.js';
+import { ObservableData } from './utilities/ObservableData.js';
 import { debounce, INPUT_DEBOUNCE_TIME } from './utilities/debounce.js';
+import { getRoleForDetector } from './utilities/getRoleForDetector.js';
 import { userPreferencesStore } from './utilities/userPreferencesStore.js';
-import { registerFrontLinkListener } from './components/common/navigation/frontLinkListener.js';
-import { EosReportModel } from './views/EosReport/EosReportModel.js';
 import { AboutModel } from './views/About/About.js';
-import { StatisticsPageModel } from './views/Statistics/StatisticsPageModel.js';
-import { LhcPeriodsModel } from './views/lhcPeriods/LhcPeriodsModel.js';
-import { HomePageModel } from './views/Home/Overview/HomePageModel.js';
 import { DataPassesModel } from './views/DataPasses/DataPassesModel.js';
-import { SimulationPassesModel } from './views/SimulationPasses/SimulationPassesModel.js';
+import { EnvironmentModel } from './views/Environments/EnvironmentModel.js';
+import { EosReportModel } from './views/EosReport/EosReportModel.js';
+import { ErrorModel } from './views/Error/ErrorModel.js';
+import Flps from './views/Flps/Flps.js';
+import { HomePageModel } from './views/Home/Overview/HomePageModel.js';
+import LhcFills from './views/LhcFills/LhcFills.js';
+import { LogsModel } from './views/Logs/LogsModel.js';
 import { QcFlagTypesModel } from './views/QcFlagTypes/QcFlagTypesModel.js';
 import { QcFlagsModel } from './views/QcFlags/QcFlagsModel.js';
-import { UserRoleSelectionModel } from './models/UserRoleSelectionModel.js';
-import { ObservableData } from './utilities/ObservableData.js';
-import { BkpRoles } from './domain/enums/BkpRoles.js';
-import { getRoleForDetector } from './utilities/getRoleForDetector.js';
-import { detectorsProvider } from './services/detectors/detectorsProvider.js';
+import RunTypeModel from './views/RunTypes/RunTypeModel.js';
+import { RunsModel } from './views/Runs/RunsModel.js';
+import { SimulationPassesModel } from './views/SimulationPasses/SimulationPassesModel.js';
+import { StatisticsPageModel } from './views/Statistics/StatisticsPageModel.js';
+import Tags from './views/Tags/Tags.js';
```

Comment:
You should try to avoid as much as possible this kind of huge changes, for example reordering imports. It adds a lot of noise for review, and someone might do the other way around in a future commit, and this is an endless cycle of changes.